### PR TITLE
changes after github org rename

### PR DESCRIPTION
**Pull request by pkliczewski**
This PR removed Parodos branding and aligns all the links changed names.

### CODE REVIEW

**Review by ElaiShalevRH — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,17 +1,17 @@
-parodos-dev.github.io 
+rhdhoperator.github.io
```
Comment (masayag) [inline README.md:1]: s/rhdhoperator/rhdhorchestrator

File: README.md [hunk 2]
```diff
@@ -23,21 +23,21 @@ This site is built by hugo static site generater and published using a github ac
 # Content Organization
 
 - content/docs \
-  The main directory for the project document 
+  The main directory for the project document
 - content/docs/workflows \
-  Document for the selected set of workflows, for https://parodos-dev/serverless-workflows
+  Document for the selected set of workflows, for https://rhdhoperator/serverless-workflows
```
Comment (masayag) [inline README.md:28]: pls change to https://rhdhorchestrator.io/serverless-workflows

File: README.md [hunk 3]
```diff
@@ -23,21 +23,21 @@ This site is built by hugo static site generater and published using a github ac
...
 - content/docs/workflows-examples \
-  Document for the examples workflows, for https://parodos-dev/serverless-workflow-examples
+  Document for the examples workflows, for https://rhdhoperator/serverless-workflow-examples
```
Comment (masayag) [inline README.md:30]: pls change to https://rhdhorchestrator.io/serverless-workflow-examples

File: README.md [hunk 4]
```diff
@@ -23,21 +23,21 @@ This site is built by hugo static site generater and published using a github ac
...
 - content/post \
   Articles, blog posts, etc.
 
 Read more the on hugo documentation https://gohugo.io/documentation/
 
 # How to add a document?
-Documents can include markdown content from all the related *`parodos-dev`* repositories. 
+Documents can include markdown content from all the related *`rhdhoperator`* repositories.
```
Comment (masayag) [inline README.md:37]: s/rhdhoperator/rhdhorchestrator

File: README.md [hunk 5]
```diff
@@ -23,21 +23,21 @@ This site is built by hugo static site generater and published using a github ac
...
 To create a document entry from a markdown file use this:
 
 ```bash
 ./generate-doc-for-repo.sh \
-    https://github.com/parodos-dev/$REPO/blob/$BRANCH//README.md > content/docs/newdoc.md
+    https://github.com/rhdhoperator/$REPO/blob/$BRANCH//README.md > content/docs/newdoc.md
```
Comment (masayag) [inline README.md:42]: s/rhdhoperator/rhdhorchestrator

File: content/1.2/docs/core-concepts/workflow-types/_index.md [hunk 6]
```diff
@@ -20,9 +20,9 @@ annotations:
 The Orchestrator plugin utilizes this metadata to facilitate the processing and visualization of infrastructure workflow inputs and outputs within the user interface.
 
 ##### Examples:
-- [Greeting](https://github.com/parodos-dev/serverless-workflows/blob/v1.2.x/greeting/greeting.sw.yaml)
-- [Ticket Escalation](https://github.com/parodos-dev/serverless-workflows/blob/v1.2.x/escalation/ticketEscalation.sw.yaml)
-- [Move2Kube](https://github.com/parodos-dev/serverless-workflows/blob/v1.2.x/move2kube/m2k.sw.yml)
+- [Greeting](https://github.com/rhdhoperator/serverless-workflows/blob/v1.2.x/greeting/greeting.sw.yaml)
```
Comment (masayag) [inline content/1.2/docs/core-concepts/workflow-types/_index.md:23]: s/rhdhoperator/rhdhorchestrator

**Review by masayag — COMMENTED** [review]
Pls `s/rhdhoperator/rhdhorchestrator` in the entire PR

**Review by masayag — APPROVED** [review]

**Pull request merged**